src/views/Dashboard/GeneticAnalyst/Home/index.vue

```diff
@@ -14,36 +14,48 @@
             view-box="0 0 120 120"
             fill
           )
-
-      DataTable(:headers="headers" :items="orderLists")
-        template(slot="prepend")
-          .ga-dashboard__text
-            h2.ga-dashboard__table-title Order Lists
-            p.ga-dashboard__table-subtitle.mb-0 List of all services ordered by Customers
-
-        template(v-slot:[`item.id`]="{ item }")
-          span {{ `${item.id.substr(0, 4)}...${item.id.substr(item.id.length - 3)}` }}
-
-        template(v-slot:[`item.action`]="{ item }")
-          .ga-dashboard__table-action.d-flex.justify-center
-            router-link(:to="{ name: 'ga-order-details', params: { id: item.id, item } }")
-              ui-debio-icon(
-                :icon="eyeIcon"
-                role="button"
-                size="16"
-                stroke
-              )
-
+      transition(name="transition-slide-x" mode="out-in")
+        .ga-dashboard__verification-status(v-if="verificationStatus !== null && verificationStatus !== 'Verified'")
+          .verification-status
+            ui-debio-icon.verification-status__icon(
+              :icon="alertIcon"
+              size="25"
+              stroke
+            )
+            .verification-status__text
+              | Your verification submission is {{ verificationStatus === 'Unverified' ? " being reviewed" : verificationStatus }} by Daogenic
+
+        DataTable(:headers="headers" :items="orderLists" v-else)
```

Comment:
I think you need to use `UiDebioDataTable` now?

Reply:
I will update to GA repo's